sql-plugin/src/main/scala/org/apache/spark/sql/rapids/complexTypeExtractors.scala

```diff
@@ -186,6 +186,99 @@ case class GpuGetMapValue(child: Expression, key: Expression)
   override def right: Expression = key
 }
 
+class GpuElementAtMeta(
+    expr: ElementAt,
+    conf: RapidsConf,
+    parent: Option[RapidsMeta[_, _, _]],
+    rule: DataFromReplacementRule)
+  extends BinaryExprMeta[ElementAt](expr, conf, parent, rule) {
+  override def convertToGpu(lhs: Expression, rhs: Expression): GpuExpression = {
+    GpuElementAt(lhs, rhs)
+  }
+}
+
+case class GpuElementAt(left: Expression, right: Expression)
```

Comment:
Seems the CPU has the third argument `failOnError`, but is skipped here. Is it OK ?

Reply:
I filed another issue https://github.com/NVIDIA/spark-rapids/issues/2272 for it's different behaviours.